Library/FilterCriteria.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recurly
+{
+    public class FilterCriteria
+    {
+        public enum Sort
+        {
+            CreatedAt,
+            UpdatedAt
+        }
+
+        public enum Order
+        {
+            Asc,
+            Desc
+        }
+
+        public Sort? SortField;
+        public Order? OrderField;
+        public DateTime? BeginTime;
+        public DateTime? EndTime;
+
+        public static FilterCriteria Instance { get { return new FilterCriteria(); } }
```

Comment:
I would add a private or internal instance constructor in addition to the static `Instance` property. That way there wouldn't be any confusion on the preferred way to create the FilterCriteria.